hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassAlertsServiceImpl.java

```diff
@@ -117,13 +128,26 @@
     @Resource
     private ManagedExecutorService executor;
 
+    public CassAlertsServiceImpl() {
+    }
+
     @PostConstruct
     public void init() {
         criteriaNoQuerySize = Integer.valueOf(properties.getProperty(CRITERIA_NO_QUERY_SIZE,
                 CRITERIA_NO_QUERY_SIZE_ENV, CRITERIA_NO_QUERY_SIZE_DEFAULT));
-    }
-
-    public CassAlertsServiceImpl() {
+        batchSize = Integer.valueOf(properties.getProperty(BATCH_SIZE,
+                BATCH_SIZE_ENV, BATCH_SIZE_DEFAULT));
+        String propTypeValue = properties.getProperty(BATCH_TYPE, BATCH_TYPE_ENV, BATCH_TYPE_DEFAULT);
+        switch (propTypeValue) {
+            case "LOGGED":
+                batchType = BatchStatement.Type.LOGGED;
+                break;
+            case "COUNTER":
+                batchType = BatchStatement.Type.COUNTER;
+                break;
+            default:
+                batchType = BatchStatement.Type.UNLOGGED;
```

Comment:
+1
These code was for testing and it makes sense to remove it in the final revision.
Good catch.